examples/service_config.rs

```diff
@@ -0,0 +1,63 @@
+#[cfg(windows)]
+extern crate windows_service;
+
+#[cfg(windows)]
+fn main() -> windows_service::Result<()> {
+    use std::ffi::OsString;
+    use std::thread;
+    use std::time::Duration;
+    use windows_service::service::{
+        ServiceAccess, ServiceErrorControl, ServiceInfo, ServiceStartType, ServiceState,
+        ServiceType,
+    };
+    use windows_service::service_manager::{ServiceManager, ServiceManagerAccess};
+
+    let manager_access = ServiceManagerAccess::CONNECT | ServiceManagerAccess::CREATE_SERVICE;
+    let service_manager = ServiceManager::local_computer(None::<&str>, manager_access)?;
+
+    // This example installs the service defined in `examples/ping_service.rs`.
+    // In the real world code you would set the executable path to point to your own binary
+    // that implements windows service.
+    let service_binary_path = ::std::env::current_exe()
+        .unwrap()
+        .with_file_name("ping_service.exe");
+
+    let service_info = ServiceInfo {
```

Comment:
There is a lot of duplication from other example. Perhaps we could query some standard windows service instead?

Reply:
i rewrote the example to either take an arg or look at the `NetLogon` service